microsoft-365/managed-desktop/service-description/DeviceRegistration.md

```diff
@@ -70,12 +62,32 @@ Your file needs to include the **exact same column headings** as the sample one
   ```
 >Failure to change any the sample data will result in your registration being rejected.   
 
+>[!TIP
+>If you're working with a brand new device, you can obtain the hardware hash without going through the whole out-of-box experience by following these steps:
+
+1. On a different device, insert a USB drive.
```

Comment:
For the powershell blocks do we want to use "```"?

Reply:
Typically for single (inline) PS, we just use the single backtick. The ``` is for whole blocks or scripts.

Reply:
rgr that